README.md

```diff
@@ -22,64 +22,23 @@ should make sure you choose strong passwords for `AUTH_CREDENTIALS`.
 
 ## Installation
 
-To run as an app on Aptible:
+To deploy Kibana as an App on Enclave:
 
- 1. Create an app in your [Aptible dashboard](https://dashboard.aptible.com) for Kibana. In the
-    steps that follow, we'll use &lt;YOUR_KIBANA_APP_HANDLE&gt; anywhere that you should substitute the
-    actual app handle the results from this step in the instructions.
+   aptible apps:create "$HANDLE" 
+   aptible deploy --app "$HANDLE" --docker-image aptible/kibana AUTH_CREDENTIALS=username:password DATABASE_URL=http://user:password@example.com
 
- 2. Use the [Aptible CLI](https://github.com/aptible/aptible-cli) to set AUTH_CREDENTIALS to the
-    username/password you want to use to access the app. To set the user to "foo" and password
-    to "bar", run:
+Note: your Elasticsearch database must be reachable at `DATABASE_URL` when this image is deployed, and the Kibana version must be compatible with the Elasticsearch version you are connecting to.
```

Comment:
Telling the user *what* should be is useful, but telling them *how* to do it is more useful:

- For the URL, tell them it's their Aptible connection URL(I noted this above)
- For the compatibility issue, tell them to look at the list below.